src/algebra/lie/subalgebra.lean

```diff
@@ -130,50 +130,59 @@ def lie_subalgebra.incl (L' : lie_subalgebra R L) : L' →ₗ⁅R⁆ L :=
 { map_lie' := λ x y, by { rw [linear_map.to_fun_eq_coe, submodule.subtype_apply], refl, },
   ..L'.to_submodule.subtype }
 
+namespace lie_hom
+
 /-- The range of a morphism of Lie algebras is a Lie subalgebra. -/
-def lie_hom.range : lie_subalgebra R L₂ :=
+def range : lie_subalgebra R L₂ :=
 { lie_mem' := λ x y,
     show x ∈ f.to_linear_map.range → y ∈ f.to_linear_map.range → ⁅x, y⁆ ∈ f.to_linear_map.range,
     by { repeat { rw linear_map.mem_range }, rintros ⟨x', hx⟩ ⟨y', hy⟩, refine ⟨⁅x', y'⁆, _⟩,
-         rw [←hx, ←hy], change f ⁅x', y'⁆ = ⁅f x', f y'⁆, rw lie_hom.map_lie, },
-  ..f.to_linear_map.range }
+         rw [←hx, ←hy], change f ⁅x', y'⁆ = ⁅f x', f y'⁆, rw map_lie, },
+  ..(f : L →ₗ[R] L₂).range }
 
-@[simp] lemma lie_hom.range_bracket (x y : f.range) :
+@[simp] lemma range_bracket (x y : f.range) :
   (↑⁅x, y⁆ : L₂) = ⁅(↑x : L₂), ↑y⁆ := rfl
```

Comment:
Shouldn't this lemma be called `lie_subalgebra.coe_bracket`? It applies to all subalgebras, not just `range`,  doesn't it? Or am I wrong?

Reply:
Good call; you're absolutely right of course!

Note that I'm not adding this lemma here. It's very old; I suspect I added it because I was having trouble getting `coe_bracket` to fire back when I didn't understand things so well.

In any case, I'll kill it now and adjust accordingly.